pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go

```diff
@@ -90,19 +96,24 @@ func (cb *CollectorBundle) prepareCollectors() {
 // addCollectorFromConfig appends a collector to the bundle based on the
 // collector name specified in the check configuration.
 //
+// ## Normal Groups
 // The following configuration keys are accepted:
 //   - <collector_name> (e.g "cronjobs")
 //   - <apigroup_and_version>/<collector_name> (e.g. "batch/v1/cronjobs")
 //
-// Note that in the versionless case the collector version that'll be used is
-// the one declared as the default version in the inventory.
+// ## CRD
+// CRDs are handled special with the crd prefix
+//   - crd/<apigroup_and_version>/<collector_name> (e.g. "crd/datadoghq.com/v1alpha1/DatadogMetric")
+// Once CRDs are collected the agent will collect the CRD and the related CR
 func (cb *CollectorBundle) addCollectorFromConfig(collectorName string) {
 	var (
 		collector collectors.Collector
 		err       error
 	)
 
-	if idx := strings.LastIndex(collectorName, "/"); idx != -1 {
+	if strings.HasPrefix(collectorName, "crd/") {
```

Comment:
IMO a follow-up PR making sure that once we use the collector config + crd we automatic collect our default set of resources from the inventory by running the collection. 

The idea would be something like below:
```
var wholeInventory 
if config.activated(ns, "crd"):
  collectorConfigInventory := addCollectorFromConfig()
  defaultInventory :=  discoverCollectorsFromDefaultInventory()
  wholeInventory  = mergeInventory(collectorConfigInventory, defaultInventory)
```